test/ProductControllerTest.scala

```diff
@@ -7,21 +7,30 @@ import play.api.mvc._
 import play.api.test._
 import play.api.test.Helpers._
 
-import java.time.LocalDateTime
-
 /** A test class for ProductController to see whether it returns the correct signal
  *
  */
 class ProductControllerTest extends PlaySpec with Results {
 
-  "Example Page#index" should {
-    "should be valid" in {
+  "getStatus() test" should {
+    "be valid" in {
       val environment = Environment.simple()
 
       val controller             = new ProductController(Helpers.stubControllerComponents(), environment)
       val result: Future[Result] = controller.getStatus().apply(FakeRequest())
 
-      result.value.get.get.toString() mustBe "Result(200, TreeMap())"
+      status(result) mustBe 200
+    }
+  }
+
+  "getInventory() test" should {
```

Comment:
the naming of the test should reflect the endpoint called. i.e "/rest/bakery/products" should be the message